indra/assemblers/html/template.html

```diff
@@ -9,7 +9,7 @@
     <meta name="author" content="">
     <link rel="icon" href="../../favicon.ico">
 
-    <title>Results Browser</title>
+    <title>INDRA DB Query Results</title>
```

Comment:
I think since this is a general assembler, it ought to be more agnostic to the source of Statements it displays so in that sense Results Browser was not bad. Maybe the title of the page could be an argument to the assembler or make_model to allow customization.